charter.md

```diff
@@ -2,92 +2,82 @@
-The intent is also to apply lessons learned from the development of [W3C's 'Securing Verifiable Credentials using JOSE and COSE' specification](https://www.w3.org/TR/vc-jose-cose), and extend them to non JSON-LD based payloads, specifically CWT Claim Sets, or other CBOR data structures.
+The intent is to apply lessons learned from the development of W3C's 'Securing Verifiable Credentials using JOSE and COSE' specification
+(https://www.w3.org/TR/vc-jose-cose), and extend them to non JSON-LD based payloads, specifically JOSE or other CBOR data structures.
 
 A short list of lessons learned:
-1. Expressive data models, such as RDF, are not necessarily suitable for
-   all use-cases outside W3C
-2. Reusing existing semantics that fit the domain well, such as provided by
-   the JWT/CWT Claims registry, provide can improve interoperability,
-   significantly
-3. Focussing on crisp technical specifications and producing separate
-   informative guidance documents helps to keep technical interested parties
-   involved
-4. Compact encoding matters and aligning JSON and CBOR provides
-   the basis for simpler interoperability and smaller specification
-
-### Out-of-Scope
-
-The working group will NOT address transport protocols, such as those developed at OAUTH, OIDF, W3C or ISO in the scope of its initial charter.
-The working group will NOT address specific credential use cases, such as "personal credentials" or "software supply chain".
-The SPICE WG's work items will not require nor forbid the use of JSON-LD. Retaining semantic interchangeability is not in-scope. Newly registered claims will have broader applicability, to both JWT and CWT use cases.
+
+1. Expressive data models, such as RDF, are not necessarily suitable for all use-cases outside W3C
+1. Reusing existing semantics that fit the domain well, such as provided by the IANA registry, provide can improve interoperability, significantly
+1. Focussing on crisp technical specifications and producing separate informative guidance documents helps to keep technical interested parties involved
+1. Compact encoding matters and aligning JSON and CBOR may provide the basis for simpler interoperability and smaller specification
+
+## Out-of-Scope
+
+The working group will NOT address specific credential use cases, such as "software supply chain".
+Retaining semantic interchangeability is not in-scope.
 
 ## Deliverables
 
 Documents produced by the working group will include the following:
 
 - An architecture document that defines the terminology (e.g., Issuer, Holder, and Verifier), the communication patterns between these parties, and the security/privacy properties.
+- A meta-data discovery document enabling holders and verifiers to discover supported protocols and formats for keys, claims, credentials and proofs.
```

Comment:
```suggestion
- A meta-data discovery document enabling the 3 roles (issuers, holders and verifiers) to discover supported protocols and formats for keys, claims, credentials and proofs.
```

Reply:
I think I understand what Denis is getting at, which is that Verifiers and Holder will need to discover metadata, but he is asserting that an Issuer will not. He may be correct, but it feels like an unnecessary limitation.
E.g., what if the issuer is a website and the browser is serving as the Holder wallet? It may be the most efficient protocol for the Issuer to discover metadata about the Holder's capabilities rather than the other way around.